ods_ci/run_robot_test.sh

```diff
@@ -324,6 +324,7 @@ if [[ ${SKIP_INSTALL} -eq 0 ]]; then
   if [[ -d "${virtenv}" ]]; then
     echo "Using a pre-created virtual environment in '${virtenv}' for poetry to save time."
     poetry config --local virtualenvs.in-project true
+    poetry config installer.parallel false
```

Comment:
Not sure, but maybe you have to add `--local` too like in the line before

Reply:
In the original post https://github.com/python-poetry/poetry/issues/3219 I did not see any recommendation to use `--local` too, but I can try for the consistency, thanks!

Reply:
Works as well, updated.